src/coreComponents/common/Logger.hpp

```diff
@@ -432,20 +432,23 @@
     }                                                                          \
   } while( false )
 
+//[DEPRECATED]
```

Comment:
add this kind of tags in the doxygen documentation
```suggestion
 * @deprecated will be replaced by ...
```